CODEOWNERS

```diff
@@ -27,3 +27,6 @@
 
 /website/content/                         @taoism4504
 /website/content/docs/plugin-portal.mdx   @taoism4504  @acahn
+
```

Comment:
Might want to add a comment explaining the connection to these UI files and motivation for notification.

Reply:
Good idea. Done in c6400e8.